libs/components/forms/src/lib/modules/file-attachment/file-size.pipe.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('File size pipe', () => {
   let decimalPipe: DecimalPipe;
 
   function validateFormatted(
-    value: number,
+    value: number | undefined | null,
```

Comment:
Hm, by making this `undefined`, we're putting two required parameters after it. Possible to rethink?

Reply:
It's in a spec, so we can let this one slide. 😄